Jenkinsfile

```diff
@@ -53,7 +53,7 @@ pipeline {
                     mkdir sample0
                     cp -r $HOME/data/testing/${env.JOB_NAME}/PangopangoF29_HYP01-10_S1244/* sample0/
                     echo "[ Run test now ]"
-                    pytest -vs --ignore=testing/test_manual_install --ignore-glob="scripts/*" && pytest --black --ignore=testing; 
+                    pytest -vs '--ignore-glob=verification/*' && pytest --black --ignore=workflow/automation/tests;
```

Comment:
Why was this added to the pytest ignore?

Reply:
I just changed the paths.

`--ignore=testing;  vs --ignore=workflow/automation/tests;`
Before
<img width="841" alt="Screen Shot 2021-12-17 at 9 46 07 AM" src="https://user-images.githubusercontent.com/466989/146446655-c209b6df-16a7-4b35-9895-5cfa88c062ce.png">

After
<img width="701" alt="Screen Shot 2021-12-17 at 9 46 24 AM" src="https://user-images.githubusercontent.com/466989/146446679-ca15a454-3844-41e6-833a-d90ec94b0383.png">


`--ignore-glob="scripts/*" vs '--ignore-glob=verification/*'`

Before
<img width="838" alt="Screen Shot 2021-12-17 at 9 48 55 AM" src="https://user-images.githubusercontent.com/466989/146446977-3162dfdb-7f19-49ca-9dfc-66585f2b0db2.png">

After
<img width="857" alt="Screen Shot 2021-12-17 at 9 50 22 AM" src="https://user-images.githubusercontent.com/466989/146447148-ef15ab38-a0b2-493d-b6c5-b3d9d0d43383.png">

and `--ignore=testing/test_manual_install` has been removed.